hw/top_earlgrey/dv/top_earlgrey_sim_cfgs.hjson

```diff
@@ -11,7 +11,7 @@
                 "{proj_root}/hw/data/common_project_cfg.hjson"]
 
   use_cfgs: ["{proj_root}/hw/ip/aes/dv/aes_sim_cfg.hjson",
-             "{proj_root}/hw/ip/entropy_src/dv/entropy_src_sim_cfg.hjson",
+//             "{proj_root}/hw/ip/entropy_src/dv/entropy_src_sim_cfg.hjson",
```

Comment:
is this intended? if this is temporary, consider adding a `// TODO` comment here.

Reply:
@senelson7 any comments?

Reply:
I believe it was commented-out because 3004 would break the entropy_src_sanity_test. So yes, it's temporary and should have a "TODO".